lab01-activity.Rmd

```diff
@@ -152,7 +158,11 @@ Be careful here with the distinction between `>` and `>>`. The code should only
 
 `cat ~/.bash_profile`
 
-**DELETE THIS TEXT AND PUT YOUR NEW CODE CHUNK HERE**
+```{bash}
+echo "source ~/.bashrc" > ~/.bash_profile
+echo "DATA=/local-scratch/course_files/MBB243/" >> ~/.bash_profile
+cat ~/.bashrc
```

Comment:
If you want to see the file you need to run this code in the terminal
cat ~/.bash_profile